optaplanner-benchmark/src/test/resources/org/optaplanner/benchmark/impl/testdataInputSolutionFile.xml

```diff
@@ -0,0 +1,8 @@
+<TestdataSolution id="1">
```

Comment:
Where is the XStream alias that makes sure this is "TestdataSolution" instead of "org.optaplanner.core.impl.testdata.domain.TestdataSolution"?

I am not sure about these test xml... it could create a maintenance burden. Normally I use the example datasets xml for integration tests and for unit tests just generate the domain model at runtime.

Of course, having a test is more important than perfecting it.